xil-Town05/docker-compose.yml

```diff
@@ -15,8 +15,6 @@ services:
       - /opt/carma-simulation/logs:/opt/carma-simulation/logs
       - ./carla-recorder/:/app/scenario_runner/metrics/data/
       - ./cdasim/logback.xml:/opt/carma-simulation/etc/logback.xml
-      - ./sumo_background_traffic/Town05.rou.xml:/opt/carma-simulation/scenarios/Town05/sumo/Town05.rou.xml
```

Comment:
Is there any reason why we removed them? I touched based with Palash about that few days ago and he mentioned that @paulbourelly999 asked him to do so that we can change the SUMO vehicle route file based on the requirements and don't need to change CDASim each time

Reply:
No i don't think so. Removing this does not stop background traffic it just switches the type of background traffic currently. We are working with Palash to have CDASim have 1 single vehicle as background traffic just for sumo to run and then have both random traffic generation tools included in carma-config.

Reply:
This got removed by mistake, i can add this back

Reply:
Seems like some of these changes are required for normal operation in this scenario. Created another PR in cdasim to build sumo cfg changes to cdasim https://github.com/usdot-fhwa-stol/cdasim/pull/243.